StabilityMatrix.Core/Models/Packages/FluxGym.cs

```diff
@@ -0,0 +1,178 @@
+using StabilityMatrix.Core.Models.FileInterfaces;
+using StabilityMatrix.Core.Models.Progress;
+using StabilityMatrix.Core.Processes;
+using StabilityMatrix.Core.Python;
+using StabilityMatrix.Core.Services;
+
+namespace StabilityMatrix.Core.Models.Packages;
+
+[Singleton(typeof(BasePackage))]
+public class FluxGym(
+    IGithubApiCache githubApi,
+    ISettingsManager settingsManager,
+    IDownloadService downloadService,
+    IPrerequisiteHelper prerequisiteHelper
+)
+    : BaseGitPackage(githubApi, settingsManager, downloadService, prerequisiteHelper),
+        ISharedFolderLayoutPackage
+{
+    public override string Name => "FluxGym";
+    public override string DisplayName { get; set; } = "FluxGym";
+    public override string Author => "cocktailpeanut";
+
+    public override string Blurb => "Dead simple FLUX LoRA training UI with LOW VRAM support";
+
+    public override string LicenseType => "N/A";
+    public override string LicenseUrl => "";
+    public override string LaunchCommand => "app.py";
+
+    public override Uri PreviewImageUri =>
+        new("https://raw.githubusercontent.com/cocktailpeanut/fluxgym/main/screenshot.png");
+
+    public override List<LaunchOptionDefinition> LaunchOptions => [LaunchOptionDefinition.Extras];
+
+    public override SharedFolderMethod RecommendedSharedFolderMethod => SharedFolderMethod.Configuration;
+
+    public override IEnumerable<SharedFolderMethod> AvailableSharedFolderMethods =>
+        new[] { SharedFolderMethod.Symlink, SharedFolderMethod.Configuration, SharedFolderMethod.None };
+
+    public override Dictionary<SharedFolderType, IReadOnlyList<string>> SharedFolders =>
+        ((ISharedFolderLayoutPackage)this).LegacySharedFolders;
+
+    public virtual SharedFolderLayout SharedFolderLayout =>
+        new()
+        {
+            RelativeConfigPath = "config.txt",
+            Rules =
+            [
+                new SharedFolderLayoutRule
+                {
+                    SourceTypes = [SharedFolderType.CLIP],
+                    TargetRelativePaths = ["models/clip"]
+                },
+                new SharedFolderLayoutRule
+                {
+                    SourceTypes = [SharedFolderType.Unet],
+                    TargetRelativePaths = ["models/unet"]
+                },
+                new SharedFolderLayoutRule
+                {
+                    SourceTypes = [SharedFolderType.VAE],
+                    TargetRelativePaths = ["models/vae"],
+                    ConfigDocumentPaths = ["path_vae"]
+                },
+                new SharedFolderLayoutRule
+                {
+                    TargetRelativePaths = [OutputFolderName],
+                    ConfigDocumentPaths = ["path_outputs"]
+                }
+            ]
+        };
+
+    public override Dictionary<SharedOutputType, IReadOnlyList<string>> SharedOutputFolders =>
+        new() { [SharedOutputType.Text2Img] = new[] { "outputs" } };
```

Comment:
This is for training, so outputs would models? Or does this handle Inference as well?

If the outputs are models, probably don't want to link them to the image output folder :)

Reply:
Good catch. Ill change that